posthog/temporal/data_imports/pipelines/pipeline/pipeline.py

```diff
@@ -249,6 +254,7 @@ def _process_pa_table(self, pa_table: pa.Table, index: int):
                 self._schema.update_incremental_field_last_value(self._last_incremental_field_value)
 
         _update_job_row_count(self._job.id, pa_table.num_rows, self._logger)
+        decrement_rows(self._job.team_id, self._schema.id, pa_table.num_rows)
```

Comment:
was wondering if there was any way to not need to increment/decrement the count and just consider the rows for this sync + rows prior to this sync? Perhaps this would make the logic simpler and more maintainable?